build/php-cs-fixer.php

```diff
@@ -3,7 +3,10 @@
 declare(strict_types=1);
 
 return (new PhpCsFixer\Config())
-    ->setFinder(PhpCsFixer\Finder::create()->in(__DIR__ . '/..'))
+    ->setFinder(PhpCsFixer\Finder::create()
+        ->in(__DIR__ . '/..')
+        // https://github.com/phpseclib/phpseclib/pull/2016#discussion_r1691282898
+        ->notPath('phpseclib/Crypt/Blowfish.php'))
```

Comment:
I think we should ignore it for now.

Reply:
tbh if we're going to be ignoring files I think it'd better just to drop PhpCsFixer all together.

In reviewing how PhpCsFixer came to be included in phpseclib...  so there was #1748 for the master branch that I cherry-picked into the 3.0 branch. Then there was #1754 but it ultimately wasn't merged because of whitespace issues similar to what we're dealing with, now. I proposed codesniffer be used instead and then we got #1760. So right now we're doing PhpCsFixer, codesniffer and pslam. Maybe we jut do codesniffer and pslam.

It'd be a shame to lose codesniffer because it does make good finds but I greatly dislike the idea of excluding entire files and of making the types of whitespace changes that it's proposing.

I suppose another idea would be to take the output of PhpCSFixer and filter out errors from lines of code that have `// @codingStandardsIgnoreStart` before it and `// @codingStandardsIgnoreEnd` after.